ClientApp/src/index.html

```diff
@@ -6,7 +6,7 @@
     <base href="/" />
 
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="icon" type="image/png" href="assets/cuttlefish_logo.png" />
```

Comment:
As discussed, for the favicon it needs to be a .ico file

Reply:
Done